cmd/hub-net-controller-manager/main.go

```diff
@@ -161,3 +170,25 @@ func main() {
 		exitWithErrorFunc()
 	}
 }
+
+// checkCRDInstalled checks if the custom resource definition is installed
+func checkCRDInstalled(discoveryClient discovery.DiscoveryInterface, gvk schema.GroupVersionKind) error {
```

Comment:
Tried to use the same function defined here https://github.com/Azure/fleet/blob/main/pkg/utils/common.go#L415,

![image](https://github.com/user-attachments/assets/221d4512-e9bb-4cd9-834c-0044633faab7)

And it looks like my vendor directory doesn't list the required file

![image](https://github.com/user-attachments/assets/7aa0d234-a6dd-418a-b9df-834132e47b3f)

Reply:
can you try to get the utils package by using the following command?

go get go.goms.io/fleet/pkg/utils@v0.10.10

i tried it in my local and i can import this package,

	"go.goms.io/fleet/pkg/utils"